api/v1beta2/foundationdbcluster_types.go

```diff
@@ -1875,6 +1903,16 @@ func (cluster *FoundationDBCluster) GetWaitBetweenRemovalsSeconds() int {
 	return duration
 }
 
+// UseMaintenaceMode returns true if UseMaintenanceModeChecker is set.
+func (cluster *FoundationDBCluster) UseMaintenaceMode() bool {
+	return pointer.BoolDeref(cluster.Spec.AutomationOptions.MaintenanceModeOptions.UseMaintenanceModeChecker, true)
```

Comment:
I think we should disable this feature per default to preserve the current behaviour. Once we are doing a new release we can think of enabling it per default.

Reply:
Ok